bugwarrior/services/teamwork_projects.py

```diff
@@ -0,0 +1,169 @@
+            return author
+
+    def get_task_url(self):
+        return self.extra["host"] + "/#/tasks/" + str(self.record["id"])
+
+    def get_default_description(self):
+        return self.build_default_description(
+            title=self.record["content"],
+            url=self.get_task_url(),
+            number=self.record["id"],
+        )
+
+    def to_taskwarrior(self):
+        task_url = self.get_task_url()
+        description = self.record.get("content", "")
+        parent_id = self.record["parentTaskId"]
+        status = self.record["status"]
+
+        due = self.parse_date(self.record.get('due-date'))
+        created = self.parse_date(self.record.get('created-on'))
+        modified = self.parse_date(self.record.get('last-changed-on'))
+
+        end = ""
+        if str(status) in ["reopened", "new"]:
+            status = "Open"
+        else:
+            end = modified
+            status = "Closed"
+
+        return {
+            'project': self.record["project-name"],
+            'priority': self.get_priority(),
+            'due': due,
+            'entry': created,
+            'end': end,
+            'modified': modified,
+            'annotations': self.extra.get('annotations', []),
+            self.URL: task_url,
+            self.TITLE: self.record.get("content", ""),
+            self.DESCRIPTION_LONG: self.record.get("description", ""),
+            self.PROJECT_ID: int(self.record["project-id"]),
+            self.STATUS: status,
+            self.ID: int(self.record["id"]),
+        }
+
+class TeamworkService(IssueService):
+    ISSUE_CLASS = TeamworkIssue
+    CONFIG_PREFIX = 'teamwork_projects'
+
+    def __init__(self, *args, **kwargs):
+        super(TeamworkService, self).__init__(*args, **kwargs)
+        self.host = self.config.get('host', '')
+        self.token = self.config.get('token', '')
+        self.client = TeamworkClient(self.host, self.token)
+        user = self.client.authenticate()
+        self.user_id = user["account"]["userId"]
+        self.name = user["account"]["firstname"] + " " + user["account"]["lastname"]
+
+    def get_comments(self, issue):
+        if self.annotation_comments:
+            if issue.get("comments-count", 0) > 0:
+                endpoint = "/tasks/{task_id}/comments.json".format(task_id=issue["id"])
+                comments = self.client.call_api("GET", endpoint)
+                comment_list = []
+                for comment in comments["comments"]:
+                    url = self.host + "/#/tasks/" + str(issue["id"])
+                    author = "{first} {last}".format(
+                        first=comment["author-firstname"],
+                        last=comment["author-lastname"],
+                    )
+                    text = comment["body"]
+                    comment_list.append((author, text))
+                return self.build_annotations(comment_list, None)
+        return ""
```

Comment:
This should probably be `return []` so that the type matches what `build_annotations` returns.